splice_machine/src/main/java/com/splicemachine/derby/stream/function/ProjectRestrictMapFunction.java

```diff
@@ -76,7 +76,7 @@ public boolean hasNativeSparkImplementation() {
         ProjectRestrictOperation op = (ProjectRestrictOperation) operationContext.getOperation();
         if (op.projection != null) {
```

Comment:
That combination of `if` with `return false` and then `return true` could be rewritten (simpler?), e.g.

```
return op.projection == null || op.hasExpressions()
```

(or similar = that begs for some tests)

Reply:
One form is shorter.  The other form is easier to read.  No need to change.